README.md

```diff
@@ -4,6 +4,11 @@
 
 # OpenCoarrays #
 
+[![CI Build Status][build img]](https://travis-ci.org/sourceryinstitute/opencoarrays)
+[![GitHub release][release img]](https://github.com/sourceryinstitute/opencoarrays/releases/latest)
+[![GitHub license][license img]](COPYRIGHT-BSD3)
+<!-- [![Release Downloads][download img]](https://github.com/sourceryinstitute/opencoarrays/releases) -->
```

Comment:
Added badges. The image is pulled from the URL in the reference link `[___ img]` and updates based on the latest release, the current build status (on master) and the license file contestants `LICENSE` A badge with total downloads is currently disabled, because I wanted to check with @rouson if he would like this content included, and also that the download count matched his records.